firmware/Makefile

```diff
@@ -33,7 +33,7 @@ deb: all
 $(OSX_DRV):
 	@echo ""
 	@echo "Checking dependencies for driver download..."
-	@which curl xzcat cpio
+	@if [ 3 != "$(which curl cpio xzcat| wc -l)" ]; then echo -e "Missing commands, please check if your OS has curl, cpio and xzcat.\n" && exit 1; fi
```

Comment:
So, I have to agree this a little bit obfuscate. Perhaps a just additional instead use for or if.

My suggestion is:

@echo "Checking dependencies (curl, xzcat, cpio) for driver download..." 

   Indirectly they will know missing something. What do you think, @patjak .